src/main/java/com/lineate/testyourlexicon/services/GameService.java

```diff
@@ -149,6 +149,20 @@ public AnswerResponseDto userActiveGameAnswer(Long userHash,
       .build();
   }
 
+  public GameEndDto endGame(Long userHash, Long gameId) {
```

Comment:
Let's add @transactional to this method since it involves multiple operations with database